external/buildscripts/sources.txt

```diff
@@ -1,535 +1,562 @@
-mono/metadata/w32mutex.h
-mono/metadata/w32process-internals.h
-mono/metadata/w32process-unix-internals.h
-mono/metadata/w32process-win32-internals.h
-mono/metadata/w32process.h
-mono/metadata/w32semaphore.h
-mono/metadata/w32socket.h
-mono/metadata/w32socket-internals.h
-PRIVATE:
-mono/metadata/threadpool-io-epoll.c
-mono/metadata/threadpool-io-kqueue.c
-mono/metadata/threadpool-io-poll.c
-####################
-#mono sgen files
-####################
-SOURCES:
-mono/sgen/sgen-alloc.c
-mono/sgen/sgen-array-list.c
-mono/sgen/sgen-cardtable.c
-mono/sgen/sgen-debug.c
-mono/sgen/sgen-descriptor.c
-mono/sgen/sgen-gc.c
-mono/sgen/sgen-gchandles.c
-mono/sgen/sgen-gray.c
-mono/sgen/sgen-hash-table.c
-mono/sgen/sgen-internal.c
-mono/sgen/sgen-los.c
-mono/sgen/sgen-marksweep.c
-mono/sgen/sgen-memory-governor.c
-mono/sgen/sgen-nursery-allocator.c
-mono/sgen/sgen-pinning-stats.c
-mono/sgen/sgen-pinning.c
-mono/sgen/sgen-pointer-queue.c
-mono/sgen/sgen-protocol.c
-mono/sgen/sgen-qsort.c
-mono/sgen/sgen-simple-nursery.c
-mono/sgen/sgen-split-nursery.c
-mono/sgen/sgen-thread-pool.c
-mono/sgen/sgen-workers.c
-mono/sgen/sgen-fin-weak-hash.c
-mono/sgen/sgen-layout-stats.c
-HEADERS:
-mono/sgen/gc-internal-agnostic.h
-mono/sgen/sgen-archdep.h
-mono/sgen/sgen-array-list.h
-mono/sgen/sgen-cardtable.h
-mono/sgen/sgen-client.h
-mono/sgen/sgen-conf.h
-mono/sgen/sgen-copy-object.h
-mono/sgen/sgen-descriptor.h
-mono/sgen/sgen-gc.h
-mono/sgen/sgen-gray.h
-mono/sgen/sgen-hash-table.h
-mono/sgen/sgen-layout-stats.h
-mono/sgen/sgen-major-copy-object.h
-mono/sgen/sgen-marksweep-drain-gray-stack.h
-mono/sgen/sgen-memory-governor.h
-mono/sgen/sgen-minor-copy-object.h
-mono/sgen/sgen-minor-scan-object.h
-mono/sgen/sgen-pinning.h
-mono/sgen/sgen-pointer-queue.h
-mono/sgen/sgen-protocol-def.h
-mono/sgen/sgen-protocol.h
-mono/sgen/sgen-qsort.h
-mono/sgen/sgen-scan-object.h
-mono/sgen/sgen-tagged-pointer.h
-mono/sgen/sgen-thread-pool.h
-mono/sgen/sgen-workers.h
-####################
-#mono cil files
-####################
-METADATA:
-mono/cil/opcode.def
-mono/cil/tables.def
-####################
-#mono arch files
-####################
-HEADERS:
-mono/arch/amd64/amd64-codegen.h
-mono/arch/x86/x86-codegen.h
-mono/arch/arm/arm-codegen.h
-mono/arch/arm/arm_dpimacros.h
-mono/arch/arm/arm-vfp-codegen.h
-mono/arch/arm/arm_vfpmacros.h
-####################
-#mono complex files
-####################
-SOURCES:
-support/libm/complex.c
-HEADERS:
-support/libm/complex.h
-support/libm/math_private.h
-####################
-#mono mini files
-####################
-mono/mini/cfgdump.h
-mono/mini/debugger-agent.c
-mono/mini/debugger-agent.h
-mono/mini/il2cpp-c-types.h
-mono/mini/il2cpp-compat.h
-mono/mini/il2cpp-stubs.cpp
-mono/mini/jit.h
-mono/mini/mini-amd64.h
-mono/mini/mini-arch.h
-mono/mini/mini-ops.h
-mono/mini/mini-unwind.h
-mono/mini/mini-x86.h
-mono/mini/mini.h
-mono/mini/optflags-def.h
-mono/mini/patch-info.h
-mono/mini/regalloc.h
-mono/mini/seq-points.h
+####################
```

Comment:
There are line ending changes and real changes here. I'll confirm that we want these line ending changes.

Reply:
The line ending changes here are the same as on the copy_il2cpp_sources.pl file. Which is the correct line ending?